### PR TITLE
Added Contact Us page.

### PR DESCRIPTION
**Pull request by hieunguyen64**
Fixed some responsive problems.
Fixed toggler button bug.
Space between sections are more constant.

### CODE REVIEW

File: Contact-us/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    <link rel="stylesheet" href="css/contact-us.css">
+</head>
+
+<body>
+    <!--Header-->
+    <section class="header">
+        <div class="container mt-3">
+            <div class="row">
+                <div class="logo col-6 col-md-3 col-lg-2 px-0">
+                    <img src="../image/logo.jpeg" width="80%" height="100%" style="float: left;">
```
Comment (thienhophu) [inline Contact-us/contact-us.html]: folder should be named as `images`

File: Contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                </div>
+                <div class="search-container col-md-6 col-lg-7 d-none d-sm-block">
```
Comment (thienhophu) [inline Contact-us/contact-us.html]: where is the default col?

File: Contact-us/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+                        <input type="text" placeholder="Search..." name="search">
+                        <button onclick="Comingsoon()" type="submit"><i class="fa fa-search"></i></button>
+                    </form>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!--Navigation bar-->
+    <div class="container mt-3 mt-md-3 mt-xl-3" style="background-color: #6983aa;">
```
Comment (thienhophu) [inline Contact-us/contact-us.html]: Does `mt-3` is enough?

File: Contact-us/contact-us.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                </p>
+            </div>
+            <div class="mapouter col-md-6">
+                <div class="gmap_canvas"><iframe width="100%" height="300px" id="gmap_canvas" src="https://maps.google.com/maps?ll=10.773922,106.6894049&q=264A Nguyễn Thị Minh Khai&t=&z=14&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                    <a href="https://techwithlove.com/"></a>
+                </div>
+            </div>
+        </div>
+
+        <div class="row pt-md-3 pt-3">
```
Comment (thienhophu) [inline Contact-us/contact-us.html]: Does `pt-3` is enough?

File: Contact-us/contact-us.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+            <div class="showroom col-md-6">
```
Comment (thienhophu) [inline Contact-us/contact-us.html]: default col ?

File: Contact-us/contact-us.html [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+                <div class="gmap_canvas"><iframe width="100%" height="300" id="gmap_canvas" src="https://maps.google.com/maps?ll=10.8304687,106.6782603&q=2a Nguyễn Oanh&t=&z=14&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>
+                    <a href="https://techwithlove.com/"></a>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!--Footer-->
+    <section id="footer">
+        <div class="container">
```
Comment (thienhophu) [inline Contact-us/contact-us.html]: Using one `<div class="container">` for the page

File: Contact-us/css/contact-us.css [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+/*mapouter*/
+
+.mapouter {
+    position: relative;
+    text-align: right;
+    height: 300px;
+    width: 100%;
+}
+.gmap_canvas {
```
Comment (thienhophu) [inline Contact-us/css/contact-us.css:16]: extra space line

**Review by thienhophu — CHANGES_REQUESTED** [review]

File: Contact-us/contact-us.html (hunk 1 above)
Comment (hieunguyen64) [reply Contact-us/contact-us.html]: done

File: Contact-us/contact-us.html (hunk 2 above)
Comment (hieunguyen64) [reply Contact-us/contact-us.html]: added col-6 as the default col

File: Contact-us/contact-us.html (hunk 3 above)
Comment (hieunguyen64) [reply Contact-us/contact-us.html]: removed `mt-md-3` and `mt-xl-3`

File: Contact-us/contact-us.html (hunk 4 above)
Comment (hieunguyen64) [reply Contact-us/contact-us.html]: removed `pt-md-3`

File: Contact-us/contact-us.html (hunk 5 above)
Comment (hieunguyen64) [reply Contact-us/contact-us.html]: added `col-12` for each `.showroom` and `.mapouter` as default col

File: Contact-us/contact-us.html (hunk 6 above)
Comment (hieunguyen64) [reply Contact-us/contact-us.html]: done

File: Contact-us/css/contact-us.css (hunk 7 above)
Comment (hieunguyen64) [reply Contact-us/css/contact-us.css:16]: done

**Review by thienhophu — APPROVED** [review]

**Pull request merged**